Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3232,8 +3232,9 @@ public override void HighlightActElement(Act act)
                     ElementInfo elementInfo = GetElementInfoWithIWebElement(e, string.Empty);
 
                     string highlightJavascript = string.Empty;
-                    if (elementInfo.ElementType == "INPUT.CHECKBOX" || elementInfo.ElementType == "TR" || elementInfo.ElementType == "TBODY")
-                        highlightJavascript = "arguments[0].style.outline='3px dashed red'";
+                    // if (elementInfo.ElementType == "INPUT.CHECKBOX" || elementInfo.ElementType == "TR" || elementInfo.ElementType == "TBODY")
+                    if (elementInfo.ElementTypeEnum == eElementType.CheckBox || elementInfo.ElementTypeEnum == eElementType.TableItem)
+                            highlightJavascript = "arguments[0].style.outline='3px dashed red'";
                     else
```

Comment:
seems like this highlight checks better use "elementInfo.ElementType" (string) for not breaking it